web/src/components/ConsentBanner.js

```diff
@@ -0,0 +1,135 @@
+import { Button } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { Fragment, PureComponent } from 'react';
+
+let cookie = name => {
+  const cookieMap = (document.cookie || '').split(';').reduce((c, s) => {
+    const bits = s.trim().split('=');
+    if (bits.length === 2) {
+      return { ...c, [bits[0].trim()]: bits[1].trim() };
+    }
+    return c;
+  }, {});
+
+  cookie = n => cookieMap[n];
+
+  return cookieMap[name];
+};
+
+class ConsentBanner extends PureComponent {
+  state = { showDetails: false };
+
+  agreeAndContinue = () => {
+    const { onAgree } = this.props;
+    document.cookie = 'gov.cms.eapd.hasConsented=true;max-age=259200'; // 3 days
+    onAgree();
+  };
+
+  toggleDetails = () => {
+    this.setState(prev => ({ showDetails: !prev.showDetails }));
+  };
+
+  render() {
+    const { showDetails } = this.state;
+
+    const hasConsented = cookie('gov.cms.eapd.hasConsented');
+    if (hasConsented) {
+      this.agreeAndContinue();
+      return null;
+    }
+
+    return (
+      <div className="card--container">
+        <div className="ds-l-container">
+          <div className="ds-l-row card">
+            <div className="ds-l-col--1 ds-u-margin-left--auto" />
+            <div className="ds-l-col--12 ds-l-sm-col--10 ds-l-lg-col--6">
```

Comment:
This is all borrowed from the admin form card work @hbillings did.

Reply:
aww yiss!!